dense_neural_class.py

```diff
@@ -0,0 +1,28 @@
+from utils import *
+class Dense_Neural_Diy:
+    def __init__(self, input_size, hidden_layer1_size, hidden_layer2_size, output_size):
+        self.input_size = input_size
+        self.hidden_layer1_size = hidden_layer1_size
+        self.hidden_layer2_size = hidden_layer2_size
+        self.output_size = output_size
+        self.weights = None
+        # init methods
+        self.random_weights()
+
+    # initialize random weights when the object is created
```

Comment:
desnecessario, vemos que eh isso que esta fazendo. tambem nao faz mt sentido dizer onde vai ser usado, a ideia de funcoes é que sejam usadas onde necessario, se so vai usar no construtor, podia ser feito in place lá. e se precisa ficar em outro canto, o comentário tá misleading